sdk/servicebus/Microsoft.Azure.WebJobs.Extensions.ServiceBus/CHANGELOG.md

```diff
@@ -11,8 +11,6 @@
 
 - Make `ServiceBusMessageActions` thread-safe.
 
-### Other Changes
```

Comment:
@weshaggard @chidozieononiwu can we make changelog checks fail PRs when the date is set on the entry?

Reply:
the issue isn't the data being set, it is this section being empty.
Still would be great if this was caught in the CI checks rather than release.

Reply:
We have talked about that but not done it yet. Prepare-Release should fail for these cases if that gets run.

Reply:
We will do both. We will use issue https://github.com/Azure/azure-sdk/issues/3690 to track that work.